benchmarks/benchmark-ab.py

```diff
@@ -367,7 +367,8 @@ def generate_csv_output():
     artifacts['Workers'] = execution_params['workers']
     artifacts['Model'] = '[.mar]({})'.format(execution_params['url'])
     artifacts['Concurrency'] = execution_params['concurrency']
-    artifacts['Requests'] = '[input]({})'.format(execution_params['requests'])
+    artifacts['Input'] = '[input]({})'.format(execution_params['input'])
```

Comment:
could you provide test log?

Reply:
I have uploaded ts_log.log and report.md.